R/pkg/R/DataFrame.R

```diff
@@ -136,7 +136,15 @@ setMethod("schema",
 
 #' Explain
 #'
-#' Print the logical and physical Catalyst plans to the console for debugging.
+#' Print the Catalyst plans to the console for debugging.
+#'
+#' With no (or, all FALSE) options, this prints the physical
+#' plan. The options are mutually exclusive: at most one can be
+#' true.
+#'
+#' @param extended print the logical plans as well as the physical plans
+#' @param codegen print the generated code for whole-stage codegen
+#' @param cost print the optimized logical plan with operator costs
```

Comment:
the doc is there, this might have broken R build
https://github.com/apache/spark/blob/beba93c562cc98f6ebe799830f55e54d7c69fa0d/R/pkg/R/generics.R#L467

Reply:
Hm, I'm not sure what to do about that. It looks like most of the other functions in that file use either `...` or a specific parameter list, without `@param` documentation, but `explain` uses `...` with `@param` documentation for both a specific parameter and the `...`. Could you give some advice?

(Also, I don't see any failures on AppVeyor? It seems to just stop mid-log.)

Reply:
It means all the param should be added there (https://github.com/apache/spark/blob/beba93c562cc98f6ebe799830f55e54d7c69fa0d/R/pkg/R/generics.R#L467)